core/pom.xml

```diff
@@ -42,6 +42,8 @@ THE SOFTWARE.
     <stapler.version>1.243</stapler.version>
     <spring.version>2.5.6.SEC03</spring.version>
     <groovy.version>2.4.7</groovy.version>
+    <!-- Currently there is a huge number of errors => we cannot fix all of them immediately -->
+    <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
Well given that `findbugs.failOnError` is the standard property name (IIRC) in this case I endorse using the property (provided I recalled correctly)